### DISCUSSION TITLE
Allow users to edit an image directly via the Image interface

### DISCUSSION THREAD

**Discussion by Jonesus** [directus/directus#3880] [Feature Requests]
Related to https://github.com/directus/directus/issues/3869

Since `v9.0.0-rc.31` the `Image` interface no longer allows users to manipulate the image itself, instead allowing the user to edit the image file's metadata. Currently if a user wants to edit an image used by a certain collection's item, they have to navigate to the file details page, edit the file there, and then get back to editing the collection item originally in question.

Some solution ideas:
- Allow the user to choose between image editor and metadata editor when creating the interface
- Add file metadata editing button to the `File` interface instead of the `Image` interface
- Add a dropdown modal instead of four buttons side by side to the `Image` interface, so that we can have more options to choose from

**Comment by benhaynes** [comment]
Just to reiterate my point on this here:

We removed the "Edit Image" (eg: transformations) button from the interface and replaced it with an "Edit File" that takes you to the File Detail page (where the Edit Image is also an option). This is necessary since many people want/need to edit things like title, description, tags, etc of the image... and had no way to get there directly.

There are already 4 buttons (the most that fit on the interface in our current design system), so we don't have room for more options (and that might make it too complex/busy anyway). Now we're discussing more ideal ways to handle this.

**Reply by Jonesus** [reply]
Coming back to this after a while and regarding

> We removed the "Edit Image" (eg: transformations) button from the interface and replaced it with an "Edit File" that takes you to the File Detail page (where the Edit Image is also an option).

Should there be a button to edit the image in the file drawer? As currently that doesn't seem to be the case

<img src="https://user-images.githubusercontent.com/16649390/113504837-81534e80-9543-11eb-9a5c-40bee05013bc.png" width="400">

compared to navigating through the file library 

![image](https://user-images.githubusercontent.com/16649390/113504842-90d29780-9543-11eb-9cf8-c6844e4f6e05.png)

Having the edit button there would be an adequate solution, but having to check which image the user wants to edit through an item in a collection, and then going to file library and finding the same item through there seems a bit tedious especially if the number of files grows large

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
